README.md

```diff
@@ -1,8 +1,8 @@
 ## Usage
 
-Create a script called `ubuntu` that looks like:
+Create a script called `bash` that looks like:
 
     #!/bin/sh
-    docker run --rm -i -t lynxtp/ubuntu:15.10 "$@"
+    docker run --rm -i -t lynxtp/ubuntu:15.10 bash "$@"
```

Comment:
Because we must pass a command to the container now, it seems like an appropriate time to make the script name match the name of the command run in the container.